### PR TITLE
Abhishek

### PR DESCRIPTION
**Pull request by abhisheksingh75**
flex mixins general development

### CODE REVIEW

File: design/mixins.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+8.  align-self - support to cross-browser 'align-self: <value>' property.
+  
+9.  order - support to cross-browser 'order: <value>' property.
+ 
+10. flex-grow - support to cross-browser 'flex-grow: <value>' property.
+
+11. flex-shrink - support to cross-browser 'flex-shrink: <value>' property. 
+
+12. flex-basis - support to cross-browser 'flex-basis: <value>' property. 
+
```
Comment (saumitra2810) [inline design/mixins.md]: Please include a sample @include code for one of the mixins.

**Comment by saumitra2810** [comment]
Looks good otherwise! :)

File: design/mixins.md (hunk 1 above)
Comment (abhisheksingh75) [reply design/mixins.md]: Hi Saumitra, sample mixin use case has been added to document..pls review and help me to close this PR.

**Comment by saumitra1993** [comment]
Stale PR, closing.

**Pull request closed without merging**